### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by peacebefore**
Added footer with link to profy.dev website, logo and empty terms page. Also finished styling page

### CODE REVIEW

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckmadhrlk3ng80b35a5qccwi8)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] Use human readable URLs [:link:](https://profy.dev/project/reddit-timer/review-tips/human-readable-links)
- [x] Use absolute instead of relative links: `<Link to="/search" />` [:link:](https://profy.dev/project/reddit-timer/review-tips/use-absolute-links)
- [x] No components that solely copy props to their children [:link:](https://profy.dev/project/reddit-timer/review-tips/unnecessary-components)
- [x] Optimize svg files with SVGOMG [:link:](https://profy.dev/project/reddit-timer/review-tips/optimize-svg-with-svgomg)
- [x] Use react-router Link instead of HTML anchor tag [:link:](https://profy.dev/project/reddit-timer/review-tips/link-instead-of-anchor-tag)

**Styles:**
- [x] **Share styles between internal and external links** [:link:](https://profy.dev/project/reddit-timer/review-tips/shared-styles-between-internal-and-external-links)
- [x] Don't use `justify-content: space-between` here [:link:](https://profy.dev/project/reddit-timer/review-tips/no-space-between-in-footer)
- [x] **Extract shared CSS values** [:link:](https://profy.dev/project/reddit-timer/review-tips/common-css-values-in-theme)
- [x] Be careful with pixel-perfectness when using relative font sizes like `0.9em` [:link:](https://profy.dev/project/reddit-timer/review-tips/careful-with-relative-font-size)
- [x] Don't use 250px padding for the footer wrapper [:link:](https://profy.dev/project/reddit-timer/review-tips/no-250px-footer-padding)
- [x] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)

**Tests:**
- [ ] Get the logo by role with `getByRole` [:link:](https://profy.dev/project/reddit-timer/review-tips/get-logo-by-role)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/footer-713055ad-9fe5-41b2-87c0-9a434095cc14). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmaax69k3m8h0b35eyw85myf)

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckmadmft43nqo0a84r1nmtdyr)

File: src/components/header/header.styles.js [hunk 1]
```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
...
 `;
 
 export const LogoContainer = styled.div`
   display: block;
+  margin-left: 80px;
+  margin-top: 33px;
 `;
 
 export const Links = styled.div`
   font-family: 'Montserrat';
```
Comment (jkettmann) [inline src/components/header/header.styles.js:19]: You can define the font-family globally on the `body` element. Then you don't need to set it here again.

File: src/components/footer/footer.styles.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  font-size: 14px;
+`;
+
+export const LinkStyle = styled(NavLink)`
+  text-decoration: none;
+  align-self: center;
+`;
+
+export const ProfyLink = styled.a`
+  font-family: 'Montserrat';
```
Comment (jkettmann) [inline src/components/footer/footer.styles.js:35]: You can define the font-family globally on the `body` element. Then you don't need to set it here again.

File: src/components/footer/footer.styles.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  width: 0px;
+  padding-left: 7px;
+`;
+
+export const LogoStyle = styled(Logo)`
+  display: block;
+`;
+
+export const Links = styled.div`
+  font-family: 'Montserrat';
```
Comment (jkettmann) [inline src/components/footer/footer.styles.js:25]: You can define the font-family globally on the `body` element. Then you don't need to set it here again.

File: src/components/footer/footer.styles.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Logo } from '../../assets/logo.svg';
+
+export const NavContainer = styled.nav`
+  height: 125px;
+  margin-left: 250px;
+  margin-right: 250px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (jkettmann) [inline src/components/footer/footer.styles.js:11]: `space-between` doesn't work here very well. That's probably why you need `padding-left: 7px` in the component below. Using `flex: 1` on the left and right links would be a better option. You can find more info [here](https://profy.dev/project/reddit-timer/review-tips/no-space-between-in-footer)

File: src/components/footer/footer.styles.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+import { NavLink } from 'react-router-dom';
+import { ReactComponent as Logo } from '../../assets/logo.svg';
+
+export const NavContainer = styled.nav`
+  height: 125px;
+  margin-left: 250px;
+  margin-right: 250px;
```
Comment (jkettmann) [inline src/components/footer/footer.styles.js:8]: The problem with using fixed margins here is that the app is not responsive. This is what the footer looks like on small screens:

image

The fixed margin is kept even though there isn't enough space (e.g. on screens with width < 500px).

Alternatively, you can use `width: 100%` together with a `max-width` according to the designs. That way the footer will take the complete screen width by default but never exceed the width given in the designs on large screen.

File: src/components/header/header.styles.js [hunk 6]
```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
 import { NavLink } from 'react-router-dom';
 
 export const NavContainer = styled.nav`
-  height: 100px;
-  margin-left: 80px;
-  margin-right: 80px;
```
Comment (jkettmann) [inline src/components/header/header.styles.js:7]: In my opinion, this was a nicer solution compared to setting the margins on the child components. This way you have the margins in one place which makes it easier to read and understand

File: src/components/header/header.styles.js [hunk 7]
```diff
@@ -2,26 +2,30 @@ import styled from 'styled-components';
...
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
+  margin-bottom: 55px;
 `;
 
 export const LogoContainer = styled.div`
   display: block;
+  margin-left: 80px;
+  margin-top: 33px;
```
Comment (jkettmann) [inline src/components/header/header.styles.js:15]: This is to vertically align the logo? The links wrapper below does the same by using `margin-top: 30px`, right?

Ann alternative is to set a fixed height of 100px to the `NavContainer` and let `display: flex` and `align-items: center` take care of centering the elements vertically.

File: src/pages/home/home.page.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (jkettmann) [inline src/pages/home/home.page.js:1]: This looks good, but there's a separate task for this component 😉

**Pull request merged**